gslib/tests/test_perfdiag.py

```diff
@@ -43,18 +43,30 @@
 class TestPerfDiag(testcase.GsUtilIntegrationTestCase):
   """Integration tests for perfdiag command."""
 
-  # We want to test that perfdiag works both when connecting to the standard gs
-  # endpoint, and when connecting to a specific IP or host while setting the
-  # host header. For the 2nd case we resolve gs_host (normally
-  # storage.googleapis.com) to a specific IP and connect to that explicitly.
-  _gs_host = boto.config.get('Credentials', 'gs_host',
-                             boto.gs.connection.GSConnection.DefaultHost)
-  _gs_ip = socket.gethostbyname(_gs_host)
-  _custom_endpoint_flags = [
-      '-o', 'Credentials:gs_host=' + _gs_ip, '-o',
-      'Credentials:gs_host_header=' + _gs_host, '-o',
-      'Boto:https_validate_certificates=False'
-  ]
+  @classmethod
+  def setUpClass(cls):
+    super(TestPerfDiag, cls).setUpClass()
+    # We want to test that perfdiag works both when connecting to the standard
+    # gs endpoint, and when connecting to a specific IP or host while setting
+    # the host header. For the 2nd case we resolve gs_host (normally
+    # storage.googleapis.com) to a specific IP and connect to that explicitly.
+    _gs_host = boto.config.get('Credentials', 'gs_host',
+                               boto.gs.connection.GSConnection.DefaultHost)
+    _gs_ip = None
```

Comment:
I know the existing code had underscores in the variable names for _gs_ip and _gs_host, but do we need them? They look like they're just local variables.

Reply:
Good call